<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogWidget</class>
 <widget class="QDialog" name="dialogWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Message</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMenu {
    color: white;
    background-color: black;
    border: 1px solid rgb(51,153,255);
}

QMenu::item {
    background-color: transparent;
}

QMenu::item:selected {
    background-color: rgb(45,45,48);
}

QWidget#centralWidget, QWidget#dialogWidget {
   background: rgba(45,45,48, 255);
}

QLabel {
   color: white;
}

QLabel#lbl_title {
    color: rgb(160,160,165);
}

QLabel#lbl_title:!active {
    color: rgb(100,100,105);
}

QLabel#lbl_icon {
    background-image: url(resources/icon_small.png);
}

QLabel#lbl_icon:!active {
    background-image: url(resources/icon_small_inactive.png);
}

QLineEdit {
   color: white;
   border: 1px solid rgb(63,63,70);
   background: rgb(63,63,70);
   selection-color: rgb(165,206,247);
   selection-background-color: rgb(58,99,144);
}

QLineEdit:focus {
   border: 1px solid rgb(51,153,255);
}

QLineEdit:disabled {
   color: rgb(81,101,92);
   border: 1px solid rgb(67,67,70);
   background: rgb(45,45,48);
}

QCheckBox {
   color: white;
}

QCheckBox:disabled {
   color: rgb(150,150,150);
}

QCheckBox::indicator {
    width: 13px;
    height: 13px;
}

QCheckBox::indicator:unchecked {
    image: url(resources/cb_icon.png);
}

QCheckBox::indicator:unchecked:hover {
    image: url(resources/cb_icon_active.png);
}

QCheckBox::indicator:checked {
    image: url(resources/cb_icon_checked.png);
}

QCheckBox::indicator:checked:hover {
    image: url(resources/cb_icon_checked_active.png);
}

QGroupBox {
    background-color: rgb(37,37,38);
    border: 0px;
    margin-top: 0.5em;
}

QGroupBox::title {
    color: white;
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}

QPushButton {
    background-color: rgb(37,37,38);
    border: 1px solid rgb(100,100,110);
    color: rgb(100,100,110);
}

QPushButton:hover {
    background-color: rgb(37,37,38);
    border: 1px solid rgb(51,153,255);
    color: rgb(51,153,255);
}

QPushButton:pressed {
    background-color: rgb(51,153,255);
    border: 1px solid rgb(51,153,255);
    color: rgb(37,37,38);
}

QPushButton#btn_close {
    border: 0px;
    background-image: url(resources/btn_close.png);
}

QPushButton#btn_close:disabled {
    background-image: url(resources/btn_close_inactive.png);
}

QPushButton#btn_close:hover {
    background-image: url(resources/btn_close_hover.png);
}

QPushButton#btn_close:pressed {
    background-image: url(resources/btn_close_pressed.png);
}

QPushButton#btn_restore {
    border: 0px;
    background-image: url(resources/btn_restore.png);
}

QPushButton#btn_restore:disabled {
    background-image: url(resources/btn_restore_inactive.png);
}

QPushButton#btn_restore:hover {
    background-image: url(resources/btn_restore_hover.png);
}

QPushButton#btn_restore:pressed {
    background-image: url(resources/btn_restore_pressed.png);
}

QPushButton#btn_minimize {
    border: 0px;
    background-image: url(resources/btn_minimize.png);
}

QPushButton#btn_minimize:disabled {
    background-image: url(resources/btn_minimize_inactive.png);
}

QPushButton#btn_minimize:hover {
    background-image: url(resources/btn_minimize_hover.png);
}

QPushButton#btn_minimize:pressed {
    background-image: url(resources/btn_minimize_pressed.png);
}

QToolTip {
    border: 0px;
    background: rgb(14,14,15);
    color: rgb(180,180,180);
}

QTabWidget QStackedWidget &gt; QWidget {
    background-color: rgb(37,37,38);
}

QTabWidget::pane {}

QTabBar::tab {
    color: white;
    background: rgb(37,37,38);
    border: 1px solid rgb(37,37,38);
    min-width: 60px;
    padding: 6px;
}

QTabBar::tab:!selected {
    color: rgb(160,160,165);
    background: rgb(45,45,48);
}

QTabBar::tab::selected {
    color: rgb(255,67,209);
}

QTabBar::tab::first::selected {
    color: rgb(230,170,0);
}

QTabBar::tab::last::selected {
    color: rgb(206,185,128);
}</string>
  </property>
  <widget class="QLabel" name="lbl_title">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>7</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Message</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_close">
   <property name="geometry">
    <rect>
     <x>317</x>
     <y>0</y>
     <width>34</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_icon">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>6</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>333</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lbl_text">
      <property name="minimumSize">
       <size>
        <width>331</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>331</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_ok">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>26</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>26</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="projectwizard.qrc"/>
 </resources>
 <connections/>
</ui>
